geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisString.java

```diff
@@ -718,15 +709,16 @@ protected void handleSetExpiration(SetOptions options) {
   ////// methods that modify the "value" field ////////////
 
   protected void valueAppend(byte[] bytes) {
-    value.append(bytes);
-  }
-
-  protected void valueSet(ByteArrayWrapper newValue) {
-    value = newValue;
+    int initialLength = value.length;
```

Comment:
Since this is being pulled from `ByteArrayWrapper`, I think it can be deleted from that class now.